File: 0392 is-subsequence/solution.cs
public class Solution {
    public bool IsSubsequence(string s, string t) {
        int i=0;
        for(int j=0;j<t.Length;j++) {
            if(i<s.Length && s[i]==t[j]) i++;
        }
        return i>=s.Length;
    }
}

File: 0392 is-subsequence/solution.rs
impl Solution {
    pub fn is_subsequence(s: String, t: String) -> bool {
        let mut j = 0;
        let ss = s.as_bytes();

        for &b in t.as_bytes() {
            if j < ss.len() && b == ss[j] {
                j += 1;
            }
        }

        j == ss.len()
    }
}
